crates/sequencer/src/sequencer.rs

```diff
@@ -1,17 +1,23 @@
-use crate::config::SequencerConfig;
+use blockifier::block_context::BlockContext;
```

Comment:
is there a clippy fix to group all imports at once?

Reply:
I don't think these should be grouped right? My formatter usually groups them when needed.